setup.py

```diff
@@ -1,4 +1,3 @@
-#!/usr/bin/env python
```

Comment:
Right, yes, it has to be executable, too.

>  And I thought we never install it by running either ./setup.py or python setup.py manually

Well, we do upload it by running `setup.py sdist`, manually. And it's convenient to not have to type `python ...`

I'd say put the shebang back, and make it executable :)